### PR TITLE
DB migration instructions..

### PR DESCRIPTION
**Pull request by melowe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,37 @@ and
 tessera help
 ```
 
+### Constellation migration
+For installations that use the constellation configuration the following deprecated options are avialable. When creating the alias to run tessura you can provide an additional system property
```
Comment (SatpalSandhu61) [inline README.md]: Typos:
avialable -> available
tessura -> tessera

File: README.md [hunk 2]
```diff
@@ -78,6 +78,37 @@ and
 tessera help
 ```
 
+### Constellation migration
+For installations that use the constellation configuration the following deprecated options are avialable. When creating the alias to run tessura you can provide an additional system property 
+
+`alias tessera="java -Dtessera.config.legacy=true -jar /somewhere/application-${version}-app.jar"`
+
+Most of the constallation configation parameters are supported. [link](https://github.com/jpmorganchase/constellation,"Constellation docs") with the exception of the following:
```
Comment (SatpalSandhu61) [inline README.md]: Typos:
constallation -> constellation
configation -> configuration

File: README.md (hunk 1 above)
Comment (melowe) [reply README.md]: yeah. should already have pushed that

File: README.md [hunk 3]
```diff
@@ -78,6 +78,37 @@ and
 tessera help
 ```
 
+### Constellation migration
+For installations that use the constellation configuration the following deprecated options are available. When creating the alias to run tessera you can provide an additional system property 
+
+`alias tessera="java -Dtessera.config.legacy=true -jar /somewhere/application-${version}-app.jar"`
+
+Most of the constallation configation parameters are supported. [link](https://github.com/jpmorganchase/constellation,"Constellation docs") with the exception of the following:
```
Comment (SatpalSandhu61) [inline README.md:86]: Typos here still

File: README.md [hunk 4]
```diff
@@ -78,6 +78,37 @@ and
...
+
+* generatekeys - Generate keys is available using the non legacy config. 
+* storage - bdb and dir are no longer supported. While storage is still supported it now only handles sqliite, memory or any jdbc url of your choosing. As with Constellation the storage has no credentials. 
+
+#### Data migration instructions (dir or bdb stores)
+To Migrate legacy datastore to tessera there is a db migration tool, for this example we'll use tessera-data-migration as the command name.
+
+`alias tessera-data-migration="java -jar /somewhere/data-migration-${version}.jar"`
+
+To migrate from dbd databases to tessera you must first export your existing store using `db_dump`
```
Comment (prd-fox) [inline README.md:96]: dbd -> bdb

**Review by prd-fox — APPROVED** [review]

**Pull request closed without merging**